package.json

```diff
@@ -70,14 +74,16 @@
     "build": "grunt",
     "docs": "jsdoc -c docs/jsdoc.conf.json",
     "hooks": "\\cp .github/hooks/pre-push .git/hooks/pre-push",
-    "lint": "yarn lint:js && yarn lint:tests && yarn lint:styles",
+    "lint": "yarn lint:ts && yarn lint:js && yarn lint:tests && yarn lint:styles",
     "lint:js": "eslint './src/js'",
+    "lint:ts": "eslint './src/ts/**/*.ts'",
```

Comment:
see comment below about changing the existing files in place rather than a new folder, will remove the need for this command